Index: b/lib/extensions/install.py
===================================================================
--- a/lib/extensions/install.py
+++ b/lib/extensions/install.py
@@ -13,6 +13,7 @@ class Install(InkstitchExtension):
         InkstitchExtension.__init__(self)
 
     def effect(self):
+        raise RuntimeError("This feature is *not supported* in the nix version!")
         api_server = APIServer(self)
         port = api_server.start_server()
         electron = open_url("/install?port=%d" % port)
