Index: b/inkstitch.py
===================================================================
--- a/inkstitch.py
+++ b/inkstitch.py
@@ -19,8 +19,9 @@ if getattr(sys, 'frozen', None) is None:
     else:
         extensions_path = "/usr/share/inkscape/extensions"

-    sys.path.remove(extensions_path)
-    sys.path.append(extensions_path)
+    if extensions_path in sys.path:
+        sys.path.remove(extensions_path)
+        sys.path.append(extensions_path)

 from inkex import errormsg
 from lxml.etree import XMLSyntaxError
