Index: b/inkstitch.py
===================================================================
--- a/inkstitch.py
+++ b/inkstitch.py
@@ -10,6 +10,10 @@ import sys
 import traceback
 from argparse import ArgumentParser
 from io import StringIO
+import warnings
+
+# ignore all warnings
+warnings.filterwarnings('ignore')
 
 if getattr(sys, 'frozen', None) is None:
     # When running in development mode, we want to use the inkex installed by
Index: b/lib/api/server.py
===================================================================
--- a/lib/api/server.py
+++ b/lib/api/server.py
@@ -19,6 +19,9 @@ from .install import install
 from .simulator import simulator
 from .stitch_plan import stitch_plan
 
+import warnings
+warnings.filterwarnings('ignore')
+
 
 class APIServer(Thread):
     def __init__(self, *args, **kwargs):
Index: b/lib/api/simulator.py
===================================================================
--- a/lib/api/simulator.py
+++ b/lib/api/simulator.py
@@ -5,4 +5,7 @@
 
 from flask import Blueprint
 
+import warnings
+warnings.filterwarnings('ignore')
+
 simulator = Blueprint('simulator', __name__)
Index: b/lib/api/stitch_plan.py
===================================================================
--- a/lib/api/stitch_plan.py
+++ b/lib/api/stitch_plan.py
@@ -7,6 +7,9 @@ from flask import Blueprint, g, jsonify
 
 from ..stitch_plan import stitch_groups_to_stitch_plan
 
+import warnings
+warnings.filterwarnings('ignore')
+
 
 stitch_plan = Blueprint('stitch_plan', __name__)
 
Index: b/lib/extensions/print_pdf.py
===================================================================
--- a/lib/extensions/print_pdf.py
+++ b/lib/extensions/print_pdf.py
@@ -29,6 +29,9 @@ from ..svg.tags import INKSCAPE_GROUPMOD
 from ..threads import ThreadCatalog
 from .base import InkstitchExtension
 
+import warnings
+warnings.filterwarnings('ignore')
+
 
 def datetimeformat(value, format='%Y/%m/%d'):
     return value.strftime(format)
Index: b/lib/utils/json.py
===================================================================
--- a/lib/utils/json.py
+++ b/lib/utils/json.py
@@ -5,6 +5,8 @@
 
 from flask.json import JSONEncoder
 
+import warnings
+warnings.filterwarnings('ignore')
 
 class InkStitchJSONEncoder(JSONEncoder):
     """JSON encoder class that runs __json__ on an object if available.
