Index: b/electron/src/renderer/router/index.js
===================================================================
--- a/electron/src/renderer/router/index.js
+++ b/electron/src/renderer/router/index.js
@@ -14,18 +14,9 @@ Vue.use(Router)
 export default new Router({
   routes: [
     {
-      path: '/simulator',
+      path: '*',
       name: 'simulator',
       component: require('@/components/Simulator').default
-    },
-    {
-      path: '/install',
-      name: 'install',
-      component: require('@/components/InstallPalettes').default
-    },
-    {
-      path: '*',
-      redirect: '/'
     }
   ]
 })
